ts/components/conversation/Timestamp.tsx

```diff
@@ -48,14 +48,29 @@ export const Timestamp = (props: Props) => {
 
   const momentValue = moment(timestamp);
   let dateString: string = '';
+  // Set the locale for the timestamps.
+  moment.locale(osLocaleSync().replace(/_/g, '-'));
```

Comment:
I guess that's a blocking call which is a bad idea to have in the renderer side 

You should be able to get the locale required by the user (and not just the system one) using:
`
  const lang = (window.i18n as any).getLocale();
`

Reply:
That code will retrieve only the generic language, e.g. `en`, not the specific locale variant, such as `en_gb`.